docs/CONTRIBUTING.md

```diff
@@ -168,6 +168,29 @@ In case of linting errors you should be able to solve most of them by automatica
 task python:format
 ```
 
+### Configuration files formatting
+
+To keep the configurations tidy and in order we use [Prettier][prettier-website] to automatically format all YAML files in the project.
```

Comment:
The rest of this document is wrapped at 80 characters.

There is currently an inconsistency in this practice in the documentation which was caused by the content being merged from two sources. The Arduino CLI-specific documentation is all wrapped at 80, but then we migrated a bunch of documentation from the [arduino/Arduino repository's wiki](https://github.com/arduino/Arduino/wiki) for things that are now implemented in Arduino CLI. That migrated content wasn't wrapped at all. I would like to make a determination of which is the official style and then use that across all the content (ideally with automation as you have done here), but I don't know which style would win out. Personally, I get a bit annoyed by 80 character wrap, since it's sort of ridiculous in 2020, but I'm happy to use whichever the official style is. I just need guidance in what that is.

So for now, I think it's best to just follow the convention that was previously established by the surrounding content. In this case, it's wrap@80, so I recommend that.

Reply:
Prettier by default wraps at 80 but in the [docs][prettier-wrap] they recommend using an higher value like 100 or 120; I personally think 120 is fine, I've used it for Python scripts too.

We can even exclude wrapping in Markdown files if we want with [`prose-wrap`][prettier-prose-wrap].

[prettier-wrap]: https://prettier.io/docs/en/options.html#print-width
[prettier-prose-wrap]: https://prettier.io/docs/en/options.html#prose-wrap

Reply:
I also think 120 is better if there is a wrap. That convention was established for the Python here (https://github.com/arduino/arduino-cli/blob/master/.flake8#L4), but then the only Markdown that was wrapped is all at 80 (all the rest is not wrapped at all).

I've been wishing for an official Arduino Markdown style guide for consistency across all the many Markdown documents we have. There is maybe some work underway on that here:
https://arduino.slack.com/archives/C014GPRCPE0/p1594631708032900
but I don't know whether it will go so far as to specify a wrap width or not.